scripts/docs_config/introduction.rst

```diff
@@ -0,0 +1,101 @@
+==============
```

Comment:
Please add a link to README to this intro

Reply:
I have added it to the fragments where I mention it

Reply:
I meant to add a link to https://oneapi-src.github.io/unified-memory-framework/introduction.html somewhere at the top of the main README. Something like, 'Documentation available at: <link>'. I know there is an 'about' section on github with the link but IMO it's easy to miss

Reply:
done